fastlane_bot/events/utils.py

```diff
@@ -1310,6 +1350,39 @@ def get_latest_events(
     return latest_events
 
 
+def validate_max_block_fetch(
+        start_block: int,
+        replay_from_block: int,
+        alchemy_max_block_fetch: int,
+        current_block: int,
+        reorg_delay: int = 0,
+) -> Tuple[int, int or None]:
+    """
+    Validates the maximum block fetch.
+
+    Parameters
+    ----------
+    start_block : int
+        The starting block number.
+    current_block : int
+        The current block number.
+    alchemy_max_block_fetch : int
+        The maximum number of blocks to fetch.
+    reorg_delay : int
+        The reorg delay.
+
+    Returns
+    -------
+    Tuple[int, int or None]
+        The starting block number and the block number to replay from.
+
+    """
+    if current_block is not None and start_block is not None:
+        if current_block - start_block > alchemy_max_block_fetch:
+            start_block = current_block - (alchemy_max_block_fetch + reorg_delay)
+    return start_block, replay_from_block
```

Comment:
Taking `replay_from_block` as input and returning it as output seems redundant in this function.